sortedm2m_tests/models.py

```diff
@@ -3,8 +3,14 @@
 from sortedm2m.fields import SortedManyToManyField
 
 
+class m2mprint:
+
+    def __unicode__(self):
+        return unicode(self.book)
```

Comment:
yes whatever it works as long as there is an assertion in the test that checks for the result is what we expect and the assertion works with python2 and python3.